jcl/src/java.base/share/classes/jdk/internal/foreign/abi/UpcallLinker.java

```diff
@@ -27,32 +27,28 @@
 
 import java.lang.foreign.FunctionDescriptor;
 import java.lang.foreign.MemorySegment;
-/*[IF JAVA_SPEC_VERSION >= 20]*/
-import java.lang.foreign.SegmentScope;
-/*[ELSE] JAVA_SPEC_VERSION >= 20 */
-import java.lang.foreign.MemorySession;
-/*[ENDIF] JAVA_SPEC_VERSION >= 20 */
 /*[IF JAVA_SPEC_VERSION >= 21]*/
 import jdk.internal.foreign.abi.AbstractLinker.UpcallStubFactory;
-/*[ENDIF] JAVA_SPEC_VERSION >= 21 */
+import jdk.internal.foreign.abi.LinkerOptions;
+/*[ELSEIF JAVA_SPEC_VERSION == 20]*/
```

Comment:
Since this file is only applicable for Java 20+, I suggest this (and similar places) should just be `[ELSE]`:
```java
/*[ELSE] JAVA_SPEC_VERSION >= 21 */
```

Reply:
Agreed and updated against the suggestion above.